ChangeLog

```diff
@@ -17,6 +17,7 @@
  * Removed deprecated MultiFORM.setMaximumNumberOfDesignPoints/getMaximumNumberOfDesignPoints
  * Removed deprecated SubsetSampling.getNumberOfSteps
  * coupling_tools.execute: deprecated get_stderr,get_stdout for capture_output, returns CompletedProcess
+ * Deprecated Nlopt|Ceres|Bonmin|CMinpack|Ipopt|TBB|HMatrixFactory::IsAvailable in favor of PlatformInfo::HasFeature
```

Comment:
Nice feature. Could we complete with a `PlatformInfo::GetFeatures`?

Reply:
ok